tests/lib/server.py

```diff
@@ -99,7 +99,7 @@ def make_mock_server(**kwargs: Any) -> _MockServer:
 
     mock = Mock()
     app = _mock_wsgi_adapter(mock)
-    server = _make_server("localhost", 0, app=app, **kwargs)
+    server = cast(_MockServer, _make_server("localhost", 0, app=app, **kwargs))
```

Comment:
Is it not possible to do:

```
server: _MockServer = _make_server(...)
```

here? I appreciate that `_make_server` doens't have type hints yet, but when it gets them, it would be good that we don't trample on them by doing dangerous casts.